utils/project/sync.go

```diff
@@ -42,48 +41,62 @@ type (
 		RelativePath string `json:"path"`
 		Message      string `json:"msg"`
 	}
+	UploadedFile struct {
+		FilePath   string `json:"filePath"`
+		Status     string `json:"status"`
+		StatusCode int    `json:"statusCode"`
+	}
+	SyncResponse struct {
+		Status        string         `json:"status"`
+		StatusCode    int            `json:"statusCode"`
+		UploadedFiles []UploadedFile `json:"uploadedFiles"`
+	}
 )
 
-// SyncProject syncs a project with its remote deployment
-func SyncProject(c *cli.Context) *ProjectError {
+func SyncProject(c *cli.Context) (*SyncResponse, *ProjectError) {
 	projectPath := strings.TrimSpace(c.String("path"))
 	projectID := strings.TrimSpace(c.String("id"))
 	synctime := int64(c.Int("time"))
-	depID := strings.TrimSpace(c.String("d"))
 
 	_, err := os.Stat(projectPath)
 	if err != nil {
-		return &ProjectError{errBadPath, err, err.Error()}
+		return nil, &ProjectError{errBadPath, err, err.Error()}
 	}
 
-	deploymentInfo, err := deployments.GetDeploymentByID(depID)
+	// deploymentInfo, err := deployments.GetDeploymentByID(depID)
 
-	if err != nil {
-		return &ProjectError{errOpNotFound, err, err.Error()}
-	}
+	// if err != nil {
```

Comment:
This code is commented to provide a temp fix so that sync works locally. A follow up PR will fix it for remote.